R/checkers.R

```diff
@@ -211,6 +211,46 @@ assert_no_repeated_elements <- function(x, arg = "x",
   invisible()
 }
 
+#' Check a set of columns in a data frame uniquely identify
+#' data frame rows.
+#'
+#' @description
+#'  Equivalently, this checks that when grouping by the columns in question,
+#'  each group has a single entry
```

Comment:
Extra spaces (my fault I think)
```suggestion
#' Equivalently, this checks that when grouping by the columns in question,
#' each group has a single entry
```